### PR TITLE
show data urls and hide text

### PR DESCRIPTION
**Pull request by dudethatbe**
here are two big burritos in one bag:
* doesn't show the URL when an image is posted
* handles the data URL format for base 64-encoded images

I tested the data URL handling locally and it worked with one exception. The regex to recognize the format was allowing any data type. I've refined it to handle a few image formats, but haven't tested that change yet

### CODE REVIEW

**Comment by dudethatbe** [comment]
So the one thing I overlooked here is when a message containing an image has any accompanying text e.g. 
"hey look a cat http://cool.cat/not-real.jpg"
This is currently supported, but I don't know how common its usage is. Personally, I'm a fan of not showing any of the text when it detects an image. But that's because it makes the code a lot easier to read. Refactoring how the messages are displayed is probably the smarter thing to do. Open to thoughts on how we handle this case

File: app/components/chat-message.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ import { action } from '@ember/object';
 export default class ChatMessage extends Component {
   @tracked gifsEnabled = true;
   @tracked imgRegex = /https?:\/\/(?:[a-z0-9-]+\.)+[a-z]{2,6}(?:\/[^/#?]+)+\.(?:jpg|gif|png|webp)(\?.*$)*/;
+  @tracked dataRegex = /data:image\/(?:png|jpeg|webp);base64,.+/;
```
Comment (mcfiredrill) [inline app/components/chat-message.js:8]: These regex's  don't need to be tracked.

**Comment by mcfiredrill** [comment]
> "hey look a cat http://cool.cat/not-real.jpg"
This is currently supported, but I don't know how common its usage is. 

Not super common but ideally I'd like to support it...hmmm I'd like to try writing a getter that returns the message body with the image urls removed.

File: app/components/chat-message.js (hunk 1 above)
Comment (dudethatbe) [reply app/components/chat-message.js:8]: yeah i was wondering about that

**Comment by dudethatbe** [comment]
on the left is the updated version to handle the data URL. and on the right is how it looks currently. i also included text in the messages if anyone wants to do that
image

**Review by mcfiredrill — APPROVED** [review]
LGTM

**Pull request merged**